client/reader/start/main.jsx

```diff
@@ -1,20 +1,43 @@
 // External dependencies
```

Comment:
I renamed the main component to plain old `Start` to try and avoid doing this :) Including 'reader' twice in the component path seems redundant (`reader/reader-start`).

Is there any standard to reflect component folder hierarchy in class names? We're just trying to namespace the feature to Reader so it doesn't potentially conflict with styles elsewhere.